public/js/add-comment.js

```diff
@@ -0,0 +1,40 @@
+/*let createCommentBtn = document.getElementById("create-comment");
+createCommentBtn.addEventListener("click", (e) => {
+  let commentText = document.getElementById("comment-text").value.trim();
+
+  //Check comment text is not empty
+  if (commentText.length === 0) {
+    // show error
+    alert("Comment cannot be empty");
+    return;
+  }
+
+  createComment(commentText);
+});
+*/
+
+async function postComment(postId) {
+  let commentText = document.getElementById("comment-text").value.trim();
```

Comment:
const here please- unless the value of a variable is reassigned within a function it should be a const.
This is because after a function stops running all of the variables created in it will be destroyed.
Every time this function is called the variable commentText will be created from fresh and destroyed once the function has been executed.